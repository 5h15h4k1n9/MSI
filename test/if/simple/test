isPositive(7)
isPositive(0)
isPositive(-10)
max(5, 3)
max(2, 7)
max(4, 4)
exit
