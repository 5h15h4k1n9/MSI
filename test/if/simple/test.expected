(string) "positive"
(string) "zero"
(string) "negative"
(int) 5
(int) 7
(int) 4
Bye-bye!
